docs/flows/flows.puml

```diff
@@ -130,7 +131,33 @@ alt#Gold #DarkSeaGreen trace
   bitdaoadmin <-- bitdaogovernance: TxHash
 end
 
+newpage
+
+autonumber "Rewards:[00]"
+
+== Rewards ==
+
+alice -> forum: create forum proposal
+alice <- guarantor: positive/negative feedback
+alice -> rewardscontract: if (positive) create rewards program
+
+community -> community: delegate votes
+
+alice -> snapshot: submit proposal
+
+bitdaoadmin -> bitdaotreasury: if (pass) allow funds
+bitdaotreasury -> rewardscontract: transfer funds for rewards
+
+alice <-- rewardscontract: execute proposal
+
 
+bob -> forum: create forum proposal
+alice <- guarantor: positive/negative feedback
```

Comment:
question: should `alice` be `bob`?

The line above has `bob` creating a proposal